internal/btf/ext_info.go

```diff
@@ -303,14 +317,42 @@ type bpfLineInfo struct {
 	LineCol     uint32
 }
 
+func (li *LineInfo) FileName() string {
+	return li.fileName
+}
+
+func (li *LineInfo) Line() string {
+	return li.line
+}
+
+func (li *LineInfo) LineNumber() uint32 {
+	return li.lineNumber
```

Comment:
Why not `return li.lineCol << bpfLineShift` and remove `lineNumber`?

Reply:
What's the difference? These fields are unexported and should always be accessed using the getters.